repose-aggregator/components/services/atom-feed-service/atom-feed-service-impl/src/main/scala/org/openrepose/nodeservice/atomfeed/impl/AtomEntryStreamBuilder.scala

```diff
@@ -112,6 +132,10 @@ object AtomEntryStreamBuilder {
 
   case class UnrecoverableIOException(cause: Throwable) extends RuntimeException(cause)
 
+  case class ServerResponseException(message: String) extends Exception(message)
+
+  object ClientErrorException extends Exception
+
   object AuthenticationException extends Exception
```

Comment:
On second thought, the `ClientErrorException` and `AuthenticationException` are only thrown and also caught in one place each. Since there is an appropriate message logged when they are caught, do we really want to make these full [inner] Classes?